spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -250,7 +288,9 @@ object DatabaseTablesDAO {
     // Postgres: INSERT INTO test_serial (id) VALUES(DEFAULT); MySQL: INSERT INTO table (id) VALUES(NULL)
 
     for (j <- i + 1 to fieldsAndValues.size + i) {
-      val value = fieldsAndValues(fieldsAndValues.keys.toList(j - i - 1))
+      val value = fieldsAndValues(fieldsAndValues.keys.toList(j - i - 1)) match
+        case value: StringValue => value.value
+        case value: ByteArrayValue => value.value
```

Comment:
With previous changes
```suggestion
      val value = fieldsAndValues(fieldsAndValues.keys.toList(j - i - 1)).value
```